.vscode/settings.json

```diff
@@ -34,5 +34,5 @@
   "typescript.format.insertSpaceAfterOpeningAndBeforeClosingJsxExpressionBraces": true,
   "editor.formatOnSave": true,
   "typescript.tsdk": "./common/temp/node_modules/typescript/lib",
-  "tslint.autoFixOnSave": true
+  "tslint.autoFixOnSave": false
```

Comment:
There are a set of lint rules that I'm very ok with auto fixing. Like redundant blank lines and such, which are just tedious to fix. But I think a lot of tslint categories require some thought.